lib/web/apiserver.go

```diff
@@ -350,6 +350,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	h.POST("/webapi/mfa/login/finish", httplib.MakeHandler(h.mfaLoginFinish))
 	h.POST("/webapi/mfa/login/finishsession", httplib.MakeHandler(h.mfaLoginFinishSession))
 	h.GET("/webapi/mfa", h.WithAuth(h.getMFADevicesHandle))
+	h.GET("/webapi/mfa/:token", httplib.MakeHandler(h.getMFADevicesWithTokenHandle))
+	h.DELETE("/webapi/mfa/:deviceName/:token", httplib.MakeHandler(h.deleteMFADeviceHandle))
```

Comment:
Similarly as above:

```suggestion
	h.DELETE("/webapi/mfa/devices/:deviceName", httplib.MakeHandler(h.deleteMFADeviceHandle))
```

and push the token ID to a param (preferred) or to the request body